src/main/java/com/google/sps/servlets/MemesServlet.java

```diff
@@ -23,8 +23,13 @@
 @WebServlet("/memes")
 @MultipartConfig
 public class MemesServlet extends HttpServlet {
+<<<<<<< HEAD
   public final String PROJECT_ID = "spring21-sps-43";
   public final String BUCKET_NAME = "spring21-sps-43.appspot.com"
+=======
+  static final String PROJECT_ID = "spring21-sps-43";
+  static final String BUCKET_NAME = "spring21-sps-43.appspot.com";
+>>>>>>> ac2cd0ffba474995ca5e94bdfe941ce924b86195
```

Comment:
Remember to check "files changed" tab to make sure your changes are as intended :) This is an artifact from a merge conflict.
```suggestion
  public static final String PROJECT_ID = "spring21-sps-43";
  public static final String BUCKET_NAME = "spring21-sps-43.appspot.com";
```

Reply:
I was wondering what that was, first time seeing it.